marketplace_builder/views/pages/release-notes/2018-09-20.liquid

```diff
@@ -81,7 +81,9 @@ Example:
 Result after using `| json` filter for readability:
 
 ```json
+{% raw %}
 {{ context.headers.HTTP_USER_AGENT | useragent | json }}
```

Comment:
This is regression - i left it that way intentionally to show user example output that is straight from his/her browser (hence json formatting, not liquid)

Reply:
ok, it was giving some errors in logs though, plase check